src/wp-includes/blocks.php

```diff
@@ -850,10 +850,52 @@ function get_hooked_blocks() {
 	return $hooked_blocks;
 }
 
+/**
+ * Adds a list of hooked block types to an anchor block's ignored hooked block types.
+ *
+ * This function is meant for internal use only.
```

Comment:
I think we can denote this with `@internal`

Reply:
Ah, interesting -- I was unaware of `@internal`.

There are a few other functions I'd added for Block Hooks in the past that were also only meant for internal use only, and folks asked me to add this note, plus `@access private` (which I'm also adding here). Since I didn't add `@internal` to any of those, I'm inclined to omit it here as well for consistency's sake. I will however inquire about it in Core Slack 🙂